src/os/bluestore/BlueStore.h

```diff
@@ -2179,6 +2179,8 @@ class BlueStore : public ObjectStore,
   uint64_t kv_ios = 0;
   uint64_t kv_throttle_costs = 0;
 
+  bool m_bluestore_fail_eio; // fail/crash on eio
```

Comment:
nit: for the sake of uniformity could you please assign default value and remove 'm_' prefix?

Reply:
I just followed the style of filestore, while it's ok to remove the 'm_' prefix